### PR TITLE
[Spree Upgrade] Replace order.shipping_method_id in current_order_serializer

### LINKED ISSUE #2679: [Spree Upgrade] Replace order.shipping_method_id in current_order_serializer
Mimic what's in this commit current_order_serializer:
https://github.com/openfoodfoundation/openfoodnetwork/pull/1651/commits/06345e91615b93b75d9e1b1411b7da33fe1dd607

app/serializers/api/current_order_serializer.rb:
    1  class Api::CurrentOrderSerializer < ActiveModel::Serializer
    2:   attributes  ... :shipping_method_id,
PR - #2729

### PR DESCRIPTION
**Pull request by luisramos0**
#### What? Why?

Mostly copied from https://github.com/openfoodfoundation/openfoodnetwork/pull/1651/commits/06345e91615b93b75d9e1b1411b7da33fe1dd607

Closes #2679 

#### What should we test?
current order serialiser spec should be green.
Cant be tested on the frontend yet.

### CODE REVIEW

File: spec/serializers/current_order_serializer_spec.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+describe Api::CurrentOrderSerializer do
+  let(:distributor) { create(:distributor_enterprise) }
+  let(:oc) { create(:simple_order_cycle) }
+  let(:li) { create(:line_item, variant: create(:variant)) }
```
Comment (sauloperez) [inline spec/serializers/current_order_serializer_spec.rb]: could we take the chance to improve the variables names to make them a bit more communicative? I'd rather name them `order_cycle` and `line_item` respectively.

File: app/serializers/api/current_order_serializer.rb [hunk 2]
```diff
@@ -12,6 +12,10 @@ def payment_method_id
     object.payments.first.andand.payment_method_id
   end
 
+  def shipping_method_id
+    object.shipping_method.id
```
Comment (sauloperez) [inline app/serializers/api/current_order_serializer.rb]: I'm might have seen it and I don't remember but, did we add a `#shipping_method` that uses `shipments` under the hood?

Comment (luisramos0) [reply app/serializers/api/current_order_serializer.rb]: object.shipping_method takes you to the orderShippingMethod concern that uses order.shipments.

File: spec/serializers/current_order_serializer_spec.rb (hunk 1 above)
Comment (luisramos0) [reply spec/serializers/current_order_serializer_spec.rb]: :+1: done.
I have also added andand to the shipping_method_id method to make it work even if there are no shipments in the order.

**Pull request merged**